### ISSUE TITLE
`cupy.cutensor.reduction` returns wrong value for max(-a) operation

### ISSUE THREAD

**Issue by asi1024** [cupy/cupy#3038]
`cupy.cutensor.reduction` returns wrong value for max(-a) operation.

reproduction code:
```py
import cupy
from cupy import cutensor
from cupy.cuda import cutensor as cuda_cutensor

n = 4
a = cupy.arange(n * n, dtype='f').reshape(n, n)
c = cupy.zeros((n), dtype='f')

desc_a = cutensor.create_tensor_descriptor(a, cuda_cutensor.OP_NEG)
desc_c = cutensor.create_tensor_descriptor(c)

c = cutensor.reduction(
    1.0, a, desc_a, ('x', 'y'),
    1.0, c, desc_c, ('x',),
    reduce_op=cuda_cutensor.OP_MAX)

print(c)  # Expected: [ -0.  -4.  -8. -12.]
```

output:
```py
[1.1754944e-38 1.1754944e-38 1.1754944e-38 1.1754944e-38]
```

**Comment by anaruse** [comment]
Sorry for being late to respond, but this problem has been fixed in cuTENSOR 1.2.  Could you check with cuTENSOR 1.2?

Output (cuTENSOR 1.1):
```
[1.1754944e-38 1.1754944e-38 1.1754944e-38 1.1754944e-38]
```

Output (cuTENSOR 1.2):
```
[  0.  -4.  -8. -12.]
```

**Comment by asi1024** [comment]
I confirmed the problem has been fixed. Thanks!
